src/guide/using-the-cli.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Using the CLI
+type: guide
+order: 5
+---
+
+Creating and running labs on [machinelabs.ai](https://machinelabs.ai) is great. Sometimes however, we prefer to start our experiments on our local machines, or, we have an existing project that we'd like to import into MachineLabs to run it there.
+
+The MachineLabs CLI makes it easy to create new labs, importing existing projects, as well as pulling labs from the MachineLabs platform - all that without leaving your terminal!
+
+## Prerequisites
+
+Before we get started looking into how to use the CLI, make sure you have the following things installed on your machine:
+
+- [Node](https://nodejs.org/en/download/)
+- npm - Comes with Node
+- [yarn](https://yarnpkg.com/en/) - Only if that's your preferred package manager
+
+## Installation
+
+The MachineLabs CLI can be easily installed using your package manager of choice. We recommend installing it globally so you can use the `ml` command in your entire system.
+
+If you're using **npm**, run the following command to install the CLI:
+
+```
+$ npm install -g @machinelabs/cli
+```
+
+As a **yarn** user, the command would be:
+
+```
+$ yarn global add @machinelabds/cli
+```
+
+That's it! You should now have the `ml` command globally available on your machine. Feel free to verify everything went smooth by running the `ml` command like this:
+
+```
+$ ml
+```
+
+Which should give you an output that looks like this:
+
+```
+Usage: ml [options] [command]
+
+
+Options:
+
+  -V, --version  output the version number
+  -h, --help     output usage information
+
+
+Commands:
+
+  login           Login to MachineLabs
+  logout          Log the CLI out from MachineLabs
+  push [options]  Push current directory as a lab 
+  init [options]  Initialize current directory as lab
+  pull [options]  Pull existing lab to your local file system
+
+```
+
+Great! Let's create our first lab using the MachineLabs CLI!
+
+## Authentication
+
+In order to push and pull labs to and from the MachineLabs platform, we need to be authenticated at some point. The CLI offers a `login` and `logout` command respectively.
+
+#### Logging in
+
+To login into MachineLabs with the CLI, simply run
+
+```
+$ ml login
+```
+
+This will open up your default browser with a handshake screen that asks you for permission to let the CLI authenticate as you. It'll also show you a hash that should match the one that is output by the CLI. Make sure to only confirm the handshake when the hash in the browser window matches the one in your terminal.
```

Comment:
I think it's worth mentioning that the CLI will always connect to the user that is currently logged into the web interface. So if the user did not log into the web interface with their regular account, they will connect the CLI to the same anonymous user that the web interface uses which is fine but won't allow the user to run the lab.

Reply:
Good point